src/Owin.Security.OpenIdConnect.Server/OpenIdConnectServerHandler.Endpoints.cs

```diff
@@ -1625,97 +1649,235 @@ internal partial class OpenIdConnectServerHandler : AuthenticationHandler<OpenId
                     RequestType = OpenIdConnectRequestType.AuthenticationRequest
                 };
             }
-            
-            AuthenticationTicket ticket;
-            if (!string.IsNullOrEmpty(request.AccessToken)) {
-                ticket = await ReceiveAccessTokenAsync(request.AccessToken, request);
-            }
-
-            else if (!string.IsNullOrEmpty(request.IdToken)) {
-                ticket = await ReceiveIdentityTokenAsync(request.IdToken, request);
-            }
-
-            else if (!string.IsNullOrEmpty(request.GetRefreshToken())) {
-                ticket = await ReceiveRefreshTokenAsync(request.GetRefreshToken(), request);
-            }
 
-            else {
-                await SendErrorPayloadAsync(new OpenIdConnectMessage {
+            if (string.IsNullOrWhiteSpace(request.Token))
```

Comment:
FYI, this property was removed from IdentityModel 5 (for ASP.NET 5) to avoid confusion with `access_token`. We'll need a `GetToken()` extension for the ASP.NET 5 version.